File: codigos/1 - estrutura sequencial/1009 - Salario com Bonus/Main.c
#include <stdio.h>

int main() {
    double salario_fixo, receita_vendas;
    
    scanf("%*s\n%lf\n%lf", &salario_fixo, &receita_vendas);
    
    printf("TOTAL = R$ %.2lf\n", salario_fixo + 0.15 * receita_vendas);
    
    return 0;
}

File: codigos/1 - estrutura sequencial/1009 - Salario com Bonus/Main.cs
using System; 

// class URI {
    // static void Main(string[] args) {
            Console.ReadLine();

            double salario_fixo = double.Parse(Console.ReadLine());
            double receita_vendas = double.Parse(Console.ReadLine());

            string total = Math.Round(salario_fixo + 0.15 * receita_vendas, 2).ToString("F2");

            Console.WriteLine($"TOTAL = R$ {total}");
    // }
// }